tracer/src/Datadog.Trace/TracerConstants.cs

```diff
@@ -15,5 +15,6 @@ internal static class TracerConstants
         public const ulong MaxTraceId = 9_223_372_036_854_775_807;
 
         public const string AssemblyVersion = "2.18.0.0";
+        public const string AssemblySemVer = "2.18.0";
```

Comment:
As already noted, SemVer _should_ include the `-prerelease` suffix. If you actually just want the "main" part, I would probably call it something like `MajorMinorBuild` or `ThreePartVersion` (i.e. drop both the "Assembly" and the "SemVer" 😄)